core/columns_api.php

```diff
@@ -161,6 +161,8 @@ function columns_get_standard( $p_enabled_columns_only = true ) {
 
 	$t_columns['notes'] = null;
 
```

Comment:
Remove extra blank line.

Also (somewhat unrelated to this change), unless the sequence in the array is significant (didn't check), maybe consider moving these 2 assignments to line 126.